packages/sync-server/src/sync/CentralSyncManager.ts

```diff
@@ -77,13 +86,19 @@ export class CentralSyncManager {
     const sessionId = generateId();
     const startTime = new Date();
 
+    const unmarkSessionAsProcessing = await this.markSessionAsProcessing(sessionId);
     const syncSession = await this.models.syncSession.create({
       id: sessionId,
       startTime,
       lastConnectionTime: startTime,
       debugInfo,
     });
 
+    // no await as prepare session (especially the tickTockGlobalClock action) might get blocked
+    // and take a while if the central server is concurrently persisting records from another client.
+    // Client should poll for the result later.
```

Comment:
```suggestion
    // client should poll for the result later.
```
Or caps at the start of the first sentence - don't mind which, if they're consistent